odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -1309,6 +1309,19 @@ Example ##ex:
 Dynamic expressions allow assigning a calculated value to an applied
 term.
 
+If a calculated value is not acceptable for the type of the term or
+its [facets](#TypeFacets), does not meet the constraints imposed by its
+: varjson
+`$Nullable` property
```

Comment:
We typically use "member" or "name/value pair" when speaking of JSON objects, following https://datatracker.ietf.org/doc/html/rfc8259#section-4.

```suggestion
`$Nullable` member
```